kornia/augmentation/container/image.py

```diff
@@ -209,6 +217,83 @@ def __packup_output__(
             # Implicitly indicating the label cannot be optional since there is a mix aug
         return output
 
+    def get_transformation_matrix(
+        self, input: torch.Tensor, params: Optional[List[ParamItem]] = None,
+    ) -> torch.Tensor:
+        """Compute the transformation matrix according to the provided parameters."""
+        if params is None:
+            raise NotImplementedError("requires params to be provided.")
+        named_modules: Iterator[Tuple[str, nn.Module]] = self.get_forward_sequence(params)
+
+        res_mat: torch.Tensor = kornia.eye_like(3, input)
+        for (_, module), param in zip(named_modules, params):
+            if isinstance(module, (_AugmentationBase, MixAugmentationBase)):
+                mat = module.compute_transformation(input, param.data)  # type: ignore
+                res_mat = mat @ res_mat
+            elif isinstance(module, (ImageSequential,)):
+                mat = module.get_transformation_matrix(input, param.data)  # type: ignore
+                res_mat = mat @ res_mat
+        return res_mat
+
+    def is_intensity_only(self, strict: bool = True) -> bool:
+        """Check if all transformations are intensity-based.
+
+        Args:
+            strict: if strict is False, it will allow non-augmentation nn.Modules to be passed.
+                e.g. `kornia.enhance.AdjustBrightness` will be recognized as non-intensity module
+                if strict is set to True.
+
+        Note: patch processing would break the continuity of labels (e.g. bbounding boxes, masks).
+        """
+        for arg in self.children():
+            if isinstance(arg, (ImageSequential,)) and not arg.is_intensity_only(strict):
+                return False
+            elif isinstance(arg, (ImageSequential,)):
+                pass
+            elif isinstance(arg, IntensityAugmentationBase2D):
+                pass
+            elif strict:
+                return False
```

Comment:
Unnecessary `else` / `elif` used after `return`
https://deepsource.io/gh/kornia/kornia/run/3492ca94-6546-4722-8a23-f570db6834d1/python/PYL-R1705

Reply:
I cannot see why it is unnecessary. The proposed solution will break the logic.